packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -203,30 +255,44 @@ export function buildFocusBackground(props) {
 export function buildFocusColor(props) {
   props = buildPropsWithDefaultButtonStyles(props);
   if (props.color === 'marketing') {
+    if (props.variant !== 'solid' && props.isInverse) {
+      return props.theme.colors.tertiary500;
+    }
     return props.theme.colors.foundation02;
```

Comment:
Let's replace this usage of `foundation02`

Reply:
Done